app/src/main/java/edu/uc/app/parkfinder/service/ParkService.kt

```diff
@@ -33,10 +35,10 @@ class ParkService {
              */
             override fun onResponse(call: Call<ArrayList<Park>>, response: Response<ArrayList<Park>>
             ) {
-                _parks.value = response.body()
+                parks.value = response.body()
```

Comment:
Name changed to better fit naming conventions seen in the style guide
https://developer.android.com/kotlin/style-guide
The underscore before a name is used for a backing property